openquake/hazardlib/gsim/kuehn_2020.py

```diff
@@ -714,15 +828,17 @@ def compute(self, ctx: np.recarray, imts, mean, sig, tau, phi):
 
         # Get PGA on rock
         pga1100 = np.exp(get_mean_values(
-            C_PGA, self.region, trt, m_b, ctx, None))
+            C_PGA, self.region, trt, m_b, ctx, None) + get_backarc_term(trt, PGA(), ctx))
```

Comment:
As suggested for AG20, in this case, it would be better to create a separate class. See for example what we did for CAN20 here: https://github.com/gem/oq-engine/tree/master/openquake/hazardlib/gsim/can20